timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+via the public internet. It will only be accessible via your AWS VPC which has been peered
+with your Timescale Forge VPC.
+</highlight>
+
+## Setup
+Before you begin, log in to the [Timescale Forge console](https://console.forge.timescale.com/).
+
+### Create a new VPC
+In the Timescale Forge console, click `VPC` in the left navigation bar to go to the VPC
+dashboard. You can add new VPCs here for your Timescale Forge services to attach to.
+The VPCs created here are peered with your own VPC as part of the setup process.
+
+<img class="main-content__illustration" src="vpc-dashboard.png" alt="Navigate to the VPC dashboard in the console"/>
+
+Click `Create VPC`, type a name for your new VPC, and provide an IPv4 CIDR block.
+E.G., `10.0.0.0/16` or `192.168.0.0/24`.
+Make sure that the CIDR block you choose for your VPC does not overlap with the
+cloud VPC you are using to create a peering connection. If the CIDR blocks overlap,
+the peering process will fail. You can always find the CIDR block of your AWS VPC
+from the AWS console.
+
+<img class="main-content__illustration" src="create-vpc.png" alt="Create a new Forge VPC"/>
+
+<highlight type="tip">
+VPC peering can be enabled for free during your Timescale Forge trial, but you will be
+required to enter a valid payment method in order to create a VPC (even though you
+will not yet be charged for it).
+</highlight>
+
+### Create a peering connection
+When you have created a Timescale Forge VPC, you are ready to create a peering connection
+between your Forge VPC and your cloud VPC. To do this, expand the dropdown menu by clicking
+the `VPC Peering` column on your Forge VPC.
+
+<img class="main-content__illustration" src="create-peering-connection.png" alt="Expand the VPC Peering dropdown menu and enter info"/>
+
+From this menu, enter the AWS Account ID, AWS VPC ID, and AWS region of your
+cloud VPC for the new peering connection. When you have entered the correct
+information, click `Add peering connection` to begin the peering process.
+
+This process is asynchronous, and results in a peering connection sent to your
+AWS account for you to accept. This is an important safety mechanism, never
+accept a peering connection from an unknown account. All Timescale Forge
+peering connections are sent from the AWS account `142548018081`.
+
+Navigate to the AWS console's
+[Peering Connections](https://console.aws.amazon.com/vpc/home#PeeringConnections:)
+dashboard. Find the new peering connection request sent from Timescale Forge, and
+accept the request.
+
+<highlight type="tip">
+Make note of the peering connection ID (starting with `pcx-`) as it is used in the next step.
+</highlight>
+
+<img class="main-content__illustration" src="aws-accept-peering-connection.png" alt="Accept peering connection in AWS console after verifying requestor account number"/>
+
+### Network routing and security
+Once you have accepted the peering connection from the last step, the two VPCs will now
```

Comment:
```suggestion
Once you have accepted the peering connection, the two VPCs will now
```

Reply:
See below for thoughts -- we can opt to keep this as well

Reply:
Agreed. I think this is a safe change as it is still in close enough proximity from "the last step".